src/Device/index.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import { default as PT } from 'prop-types'
+import cx from 'classnames'
+import Spinner from '../Spinner'
+
+
+const Device = ({variant, url, className,busy,...props})=> {
+  const styles = cx(
+    'ola_device',
+    { [`ola_device-${variant}`]: variant },
+    { 'is-busy': busy },
+    className
+  )
+ 
+  return (
+    <div className={styles}{...props}>
+      <div className='ola_device-frame'>
+        {url ? <img
+          className={'ola_device-content'}
+          src={url} /> : busy &&
+          <div className={'ola_device-content ola_device-spinner'}>
+            <Spinner size="medium"/>
```

Comment:
`busy` property can be a string, this allows to insert a copy like "please wait".
Take a look to Button, for example: https://github.com/marketgoo/Ola/blob/development/src/Button/index.js#L25

Reply:
added too! Thanks Oscar for suggestions very useful.
I attach a screenshoot 
![Captura de pantalla 2021-04-23 a las 17 50 41](https://user-images.githubusercontent.com/60300665/115900057-cc37f600-a45f-11eb-88b8-47876dd23a08.png)